packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -20,9 +14,13 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
   return {
     root: [
       classNames.root,
-      getFocusStyle(theme, -2),
       {
-        color: semanticColors.link
+        color: semanticColors.link,
+        selectors: {
+          '.ms-Fabric--isFocusVisible &:focus': {
+            outline: `1px solid ${theme.palette.neutralSecondary}`
```

Comment:
If this approach is approved, consider adding an in-line comment here as to why we can't use `getFocusStyle` with a link to the issue.

Reply:
It is possible we could getFocusStyle but it would have to change fundamentally.

getFocusStyle will set the target as `position: relative`, and layer on absolute elements to avoid changing the layout shape.

That approach just doesn't work well with links, which can wrap multiple lines.

Comment here can be something like:

```ts
// Due to Link elements expected to wrap as inline, the getFocusStyle helper would not work here.
// For this one scenario (a focusable text-wrappable component), we will manually define the
// focus.
```